### PR TITLE
correcting minor typos

### PR DESCRIPTION
**Pull request by prince-chrismc**

### CODE REVIEW

File: best-practice-authorisation.md [hunk 1]
```diff
@@ -249,7 +248,7 @@ The access token MUST be a JSON Web Signature (JWS) as defined by [RFC 7515](RFC
 JWAs MUST NOT be used.
 
 The JWS MUST be signed with `RSASSA-PKCS1-v1_5 using SHA-512`, meaning the value of the `alg` field in the
-token's JOSE header MUST be set to `RS512` as defined in [RFC 7518](RFC-7518). An example JOSE header would
+token's JOSE header MUST be set to `RS512` as defined in [RFC 7518](RFC-7518). An example JOSN header would
```
Comment (garethsb-sony) [inline best-practice-authorisation.md]: What is "JOSN"? JOSE is JSON Object Signing and Encryption, defined by https://tools.ietf.org/html/rfc7515#section-2

File: best-practice-authorisation.md [hunk 2]
```diff
@@ -66,12 +66,11 @@ A short lived JSON Web Token that may be used by a client to access privileged r
 
 ### Bearer Token
 
-Bearer tokens are longer lived in access tokens. They are passed from the Authorization Server to the client after successful authentication of client credentials, and contain an access token. Bearer tokens can be used to retrieve further access tokens from the Authorization Server once the original access token expires.
+Bearer tokens are longer lived then access tokens. They are passed from the Authorization Server to the client after successful authentication of client credentials, and contain an access token. Bearer tokens can be used to retrieve further access tokens from the Authorization Server once the original access token expires.
```
Comment (garethsb-sony) [inline best-practice-authorisation.md]: This is also dealt with (along with several other typos) in PR #22 at https://github.com/AMWA-TV/nmos-api-security/pull/22/commits/18b7c8d5f147d284c59a49fd3be265ee13cc5e39#diff-4b56e8871126a03a6dd47de63051dfa3R69.

File: best-practice-authorisation.md (hunk 1 above)
Comment (prince-chrismc) [reply best-practice-authorisation.md]: Thank you I did not know that was a thing

**Comment by prince-chrismc** [comment]
This can be closed since there was no unfixed issue

File: best-practice-authorisation.md (hunk 1 above)
Comment (garethsb-sony) [reply best-practice-authorisation.md]: To be honest, neither did I until I reviewed this spec!

Comment (prince-chrismc) [reply best-practice-authorisation.md]: maybe add a link to it in your PR #22

Comment (garethsb-sony) [reply best-practice-authorisation.md]: Yes, that's a good plan, but I will wait to get some feedback on #22, #23, #24, #25 before going further. I suspect the authors are busy with build-up to NAB at the moment.

**Comment by garethsb-sony** [comment]
Thanks, @prince-chrismc. Hopefully, the other PR will be merged very soon.

**Pull request closed without merging**